### PR TITLE
Painful translator

### PR DESCRIPTION
**Pull request by blixit**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-XXX

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Explained my technical strategy below if feature is complex.

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn version:testing` (this will create a commit with a tag)
- then run `git push --follow-tags`

## Technical strategy

> _Insert technical strategy_

### CODE REVIEW

File: src/libs/i18n.ts [hunk 1]
```diff
@@ -19,3 +20,8 @@ export const i18n = setupI18n({
 export function _(id: MessageDescriptor): string {
   return i18n._(id)
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function __(strings: TemplateStringsArray, ...values: any[]): string {
```
Comment (blixit) [inline src/libs/i18n.ts:25]: le any vient du typage de la fonction t()

File: src/features/auth/pages/Login.tsx [hunk 2]
```diff
@@ -74,19 +73,19 @@ export const Login: FunctionComponent<Props> = function (props: Props) {
             <StyledInline>
               <Warning size={24} />
               <Typo.Caption color={ColorsEnum.ERROR}>
-                {_(t`E-mail ou mot de passe incorrect`)}
+                {__`E-mail ou mot de passe incorrect`}
```
Comment (BenStirrup) [inline src/features/auth/pages/Login.tsx:76]: J'anticipe que `lingui` ne puisse détecter les wordings de cette manière.
Il faut lancer `yarn translations:extract` pour vérifier cela.

Comment (blixit) [reply src/features/auth/pages/Login.tsx:76]: bien vu ! c'est dommage de devoir adopter cette écriture barbare et lourde :\

Comment (BenStirrup) [reply src/features/auth/pages/Login.tsx:76]: Fais remonter cela aux gens de BAM qui souhaite démocratiser `lingui` ^^

**Pull request closed without merging**